### ISSUE TITLE
Screen Name and Broadcast Titles

### ISSUE THREAD

**Issue by Joey-JorEl** [baliscope/periapi#8]
I have noticed that the the downloaded files only contain the username, timestamp, and broadcast id code. I would like it to include username, screen name, timestamp, and broadcast title. This will greatly help with sorting broadcasts for viewing. Is there a way to configure this?

**Comment by scifiak** [comment]
Yes, having the Broadcast title incorporated in the save file name, would be fantastic and a great help! ♥
